connectors/katalog/pkg/connector/connector.go

```diff
@@ -95,30 +100,36 @@ func (r *Handler) createAsset(c *gin.Context) {
 		return
 	}
 
+	errString := "Error during create asset! Error:"
+	secretName, secretNamespace, err := vault.GetKubeSecretDetailsFromVaultPath(request.Credentials)
+	if err != nil {
+		r.reportError(c, http.StatusInternalServerError, errString+err.Error())
+		return
+	}
+
 	asset := &v1alpha1.Asset{
 		ObjectMeta: v1.ObjectMeta{Namespace: request.DestinationCatalogID, Name: request.DestinationAssetID, GenerateName: FybrikAssetPrefix},
```

Comment:
What happens if DestinationAssetID is an illegal name for k8s resource?

Reply:
I think the r.client.Create command will fail